src/community/roadmap/index.md

```diff
@@ -29,26 +29,27 @@ We’ve also recently:
 - improved how the Breadcrumbs component appears on screen readers
 - fixed some alignment issues with Radio buttons and Checkboxes
 - introduced new features to help you include only the components your service uses
-- introduced a new [Password input component](/components/password-input/)
-- updated the crown in the header, favicon and social share images
-- updated the components, patterns and styles to be compliant with WCAG 2.2
-- made it easier for teams to understand [what's changed in WCAG 2.2 and what they need to do](/accessibility/wcag-2.2)
 
 ## Working on now
 
 We're:
 
-- making the guidance around form design and validation easier to find
-- working out how the Design System might need to change when used for pages embedded in apps using web views
+- making enhancements to the File upload component and the Details component
```

Comment:
Just a note that the details component enhancements won't be happening following some feedback from DAC. We'll have a writeup for this soon.